Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -525,6 +525,42 @@ public extension UIView {
 		anchorCenterXToSuperview()
 		anchorCenterYToSuperview()
 	}
+    
+    /// SwifterSwift: Blur the background of the view
+    ///
+    /// - Parameter effectStyle: The UIBlurEffectStyle to apply to the background
+    @available(iOS 9, *) public func blurBackground(using effectStyle: UIBlurEffectStyle) {
+        
+        backgroundColor = .clear
+        removeBlurs()
+        
+        let blurEffect = UIBlurEffect(style: effectStyle)
+        let blurredEffectView = UIVisualEffectView(effect: blurEffect)
+        blurredEffectView.frame = bounds
+        
+        addSubview(blurredEffectView)
+        sendSubview(toBack: blurredEffectView)
+        blurredEffectView.fillToSuperview()
```

Comment:
@drewpitchford We're trying not to use SwifterSwift extensions inside of other SwifterSwift extensions. Maybe I should add this to the contributing guidelines 🤔